ndcube/utils/wcs.py

```diff
@@ -183,6 +183,10 @@ def _wcs_slicer(wcs, missing_axes, item):
         new_wcs = wcs.slice(item_checked)
     # if it a tuple like [0:2, 0:3, 2] or [0:2, 1:3]
     elif isinstance(item, tuple):
+        # Ellipsis slicing is currently not supported.
+        # Raise an error if user tries to slice by ellipsis.
+        if type(Ellipsis) in [type(item_i) for item_i in item]:
```

Comment:
Can't you do this with:
```suggestion
        if Ellipsis in item:
```
?